dotnet-desktop-guide/framework/winforms/controls/how-to-create-an-unbound-windows-forms-datagridview-control.md

```diff
@@ -1,8 +1,6 @@
 ---
 title: How to Create an Unbound Windows Forms DataGridView Control
-
-description: Populate an unbound Windows Forms DataGridView Control programmatically and display a small amount of data in a table format without binding it to a data source
-   
+description: Populate an unbound Windows Forms DataGridView Control programmatically and display a small amount of data in a table format without binding it to a data source   
```

Comment:
So, it looks to me like there was no update done to this title because it was already done? The spreadsheet said this was a duplicate title issue. But you removed extra spaces from the metadata. Is that correct?

Reply:
Yes. No update to this Title in this topic.  To make the Titles unique in topics and avoid duplication, I have already changed the Title in the later topic unique.
Yes. Removed empty lines between Title and Description as a part of running and fixing Doc Validation errors in this repo.